package.json

```diff
@@ -1,1238 +1,1294 @@
 {
-    "name": "source-engine-support",
```

Comment:
I'm not going to merge this space to tabs conversion. Whether spaces or tabs are better is a different discussion but not for this merge request

Reply:
Sorry, this was a two default settings for JSON formater in linux vscode where when I save something that do use of this extension the entire package.json file got reformated.